docs/Makefile

```diff
@@ -12,7 +12,7 @@ BUILDDIR      = _build
 
 APISDIR       = $(shell printf "import alsdkdefs\nprint(alsdkdefs.get_apis_dir())" | python)
 APIDOCSDIR	  = ./$(BUILDDIR)/api_html
-APISPECFILES  := $(shell find $(APISDIR) -name '*.yaml')
+APISPECFILES  := $(shell find $(APISDIR) -regex ".*.v[0-9]*.yaml")
```

Comment:
Really small comment: dot `.` is a special symbol in regex, probably you want something like `\.` where appropriate

Reply:
Let fix this and the other places that have this in follow PRs